GitCommands/Git/GitSshHelpers.cs

```diff
@@ -28,14 +28,10 @@ bool DoArgumentsRequireSsh()
         public static void SetSsh(string path)
         {
             // Git will use the embedded OpenSSH ssh.exe if empty/unset
-            if (!string.IsNullOrEmpty(path) && !File.Exists(path))
-            {
-                path = "";
-            }
-
             Environment.SetEnvironmentVariable("GIT_SSH", path, EnvironmentVariableTarget.Process);
```

Comment:
I would remove also the comment.

Reply:
That comment still applies to setting the environment
(It would be easier to let GIT_SSH override the GE settings and not set the value if empty (no need to override), but that would require more changes for something not so important.)